demos/smart_classroom_demo/README.md

```diff
@@ -62,15 +64,15 @@ Options:
     -exp_r_fd                      Optional. Expand ratio for bbox before face recognition.
     -t_reid                        Optional. Cosine distance threshold between two vectors for face reidentification.
     -fg                            Optional. Path to a faces gallery in .json format.
-    -no_show                       Optional. Do not show processed video.
-    -last_frame                    Optional. Last frame number to handle in demo. If negative, handle all input video.
     -teacher_id                    Optional. ID of a teacher. You must also set a faces gallery parameter (-fg) to use it.
+    -no_show                       Optional. Do not show processed video.
     -min_ad                        Optional. Minimum action duration in seconds.
     -d_ad                          Optional. Maximum time difference between actions in seconds.
     -student_ac                    Optional. List of student actions separated by a comma.
+    -top_ac                        Optional. List of student actions (for top-k mode) separated by a comma.
     -teacher_ac                    Optional. List of teacher actions separated by a comma.
     -a_id                          Optional. Target action name.
-    -a_top                         Optional. Number of first K students. If this parameter is positive, the demo detects first K persons with the action, pointed by the parameter "a_id"
+    -a_top                         Optional. Number of first K students. If this parameter is positive, the demo detects first K persons with the action, pointed by the parameter 'top_id'
```

Comment:
This change is actually wrong - the parameter is called `-a_id`. Maybe you could fix the help string while you're at it?

Reply:
`showUsage()` was incorrect.